src/Phoenix.vue

```diff
@@ -1,27 +1,21 @@
 <template>
-  <div
-    id="Phoenix"
-    :class="['uk-flex uk-flex-column uk-height-1-1', { loginGradient: showGradientBackground}]"
-    :style="showImageBackground ? { 'background-image': 'url('+configuration.theme.logo.background+')',
```

Comment:
becasue we no longer need these configuration values please drop them from the theme jsons files as well as from the store